### DISCUSSION TITLE
MySQL

### DISCUSSION THREAD

**Discussion by amyb12345** [docker/extension-ideas#7] [Ideas]
Note: This is a transfer from our initial method of collecting new extension requests. If you have additional information on how this will help you or other functionality you'd like to see, please add more details in the comments.

**Comment by marcodeppe** [comment]
Since there is a PostgreSQL Docker extension now, having a MySQL one would be great :)
https://open.docker.com/extensions/marketplace?extensionId=mochoa/pgadmin4-docker-extension

**Comment by raydleemsc** [comment]
Can we see performance throughput statistics for secured db containers in the same vein as MySQL workbench?

**Comment by tippexs** [comment]
I think I will simply try to wrap PHPmyAdmin in a Docker Extension. This is similar what the extension for PGSQL is doing. I found some nice dark and light themes so we can respect the users preference. Have to fine tune my sessions but a POC should be doable while DockerCon. Looking forward to drop a link to a repo soon!
